sdk/tables/azure-data-tables/src/test/resources/session-records/TablesAsyncClientTest.submitTransactionAsync.json

```diff
@@ -0,0 +1,77 @@
+{
+  "networkCallRecords" : [ {
```

Comment:
Just curious why this affected recordings

Reply:
It didn't, I just ran the tests against the service in record mode to ensure the contents looked the same as what we had in our recordings already. Since it looked good, I decided to grab the new recordings instead, considering I changed the name of the test methods and all that.